src/frontend/package.json

```diff
@@ -17,12 +17,14 @@
     "@testing-library/user-event": "^12.6.3",
     "@types/file-saver": "^2.0.5",
     "@vitejs/plugin-react": "^4.0.0",
-    "axios": "^0.27.2",
+    "axios": "^1.6.5",
```

Comment:
you shouldn't have changes to any files except CreditsPage.tsx. You can fix this by running: `git checkout develop -- file/path` for each other changed file